src/data/services/LmsApiService.ts

```diff
@@ -4,6 +4,7 @@ import type { AxiosResponse } from 'axios';
 
 import { configuration } from '../../config';
 import generateFormattedStatusUrl from './apiServiceUtils';
+import { EnterpriseGroup, EnterpriseLearner, PaginatedCurrentPage } from '../../types';
```

Comment:
This line isn't needed any longer since types in the base `types.d.ts` were declared `global`, and is causing tests to fail.